applications/nrf_desktop/description.rst

```diff
@@ -952,6 +952,22 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for information abou
 
 .. nrf_desktop_fastpair_important_end
 
+nRF54L MCUboot provisioning
+===========================
+
+nRF54L-based nRF Desktop devices enable hardware cryptography for the MCUboot bootloader.
+The public key that MCUboot uses to validate the application image is securely stored in the hardware Key Management Unit (KMU).
+In this use case, the application image is automatically signed by the |NCS| build system.
+However, the public key is not automatically provisioned to the device when programming the bootloader and the application images using the ``west flash`` command.
+
+To provision the MCUboot keys, use the ``west ncs-provision`` command before programming the bootloader and application images.
+Make sure that the provisioned public key is generated from the private key that was used to sign the application image.
+The private keys are stored in the application configuration directory of the board.
+Path to the private key is defined by the ``SB_CONFIG_BOOT_SIGNATURE_KEY_FILE`` sysbuild Kconfig option.
+You only need to provision one public key to an nRF Desktop device.
+Keep in mind that after erasing the device, you must provision it again.
```

Comment:
```suggestion
.. note::
    If you are erasing the device, you must provision it again.
```
I think this could be a note since this is not part of the provisioning procedure above?, and if user erases the device, he need to provision the device again. If yes, could be added after line no:970.

Reply:
Making it as a note sounds like a good idea. Changed.

Reply:
Rephrased the note a bit to improve readability too